src/text-link.tsx

```diff
@@ -70,21 +71,30 @@ export type TextLinkProps = HrefProps | ToProps | OnPressProps;
 const TextLink: React.FC<TextLinkProps> = ({children, className = '', small, disabled, ...props}) => {
     const classes = useStyles();
     const isInverse = useIsInverseVariant();
-
     const {formStatus} = useForm();
+    const {analytics} = useTheme();
+
+    const createDefaultTrackingEvent = (): TrackingEvent => {
+        if (analytics.eventFormat === 'google-analytics-4') {
+            return {
+                name: eventNames.userInteraction,
+                component_type: 'link',
```

Comment:
button link and text link have the same type, but button component has differents types, should button link and link also have differents types?

Reply:
no, they are tracked with the same type, by DA Team definition